### PR TITLE
Adding notes to NOTES.txt

### PR DESCRIPTION
**Pull request by Ian-Noy**
Adding notes to NOTES.txt

### CODE REVIEW

File: NOTES.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@ This is a notes document.
 
 There are many notes contained in this document.
 
-Here is an additional note that is being added.
\ No newline at end of file
+Here is an additional note that is being added.
+
+Adding another note to the file.
```
Comment (fouimette) [inline NOTES.txt]: Does everything need to be this way?

Comment (fouimette) [inline NOTES.txt:7]: Here is a cool pic.

image

File: NOTES.txt [hunk 2]
```diff
@@ -2,4 +2,6 @@ This is a notes document.
 
 There are many notes contained in this document.
 
-Here is an additional note that is being added.
\ No newline at end of file
+Here is an additional note that is being added.
```
Comment (fouimette) [inline NOTES.txt:5]: Here is a comment without a pic.

File: NOTES.txt (hunk 1 above)
Comment (Ian-Noy) [reply NOTES.txt:7]: This is a cool pic.

File: NOTES.txt (hunk 2 above)
Comment (Ian-Noy) [reply NOTES.txt:5]: Sad face.  I like pics.

File: NOTES.txt [hunk 3]
```diff
@@ -2,4 +2,6 @@ This is a notes document.
...
+
```
Comment (fouimette) [inline NOTES.txt:6]: This is a blank line indicated by a plus "+" sign.

**Review by fouimette — APPROVED** [review]
This code is ready for prime time.

**Comment by Ian-Noy** [comment]
Code has been approved.  Going to push changes to master.

**Pull request closed without merging**